dell/xps/15-9570/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib, ... }:
+
+{
+  imports = [
+    ../../../common/cpu/intel
+    ../../../common/pc/laptop
+  ];
+
+  # TODO: boot loader
+  boot.loader.systemd-boot.enable = lib.mkDefault true;
```

Comment:
Is there anything inherently that would require systemd-boot over grub? If not than we should leave this up to the user to decide.